### PR TITLE
Update iOS OC doc

### PR DESCRIPTION
**Pull request by nasli**
Some minor updates of the iOS OC doc. Pending increase with new features

### CODE REVIEW

File: user_manual/ios_app.rst [hunk 1]
```diff
@@ -72,7 +72,7 @@ Sharing Files
 ~~~~~~~~~~~~~
 
 The little arrow button on the bottom left has options for sharing the file via
-email and instant messaging, saving or copying, and printing. 
+email, instant messaging or any other of your apps, saving or copying.
```
Comment (settermjd) [inline user_manual/ios_app.rst]: This segment of the sentence is slightly confusing. I suggest writing it as "email, instant messaging (or any other of your apps), as well as saving and copying."

File: user_manual/ios_app.rst [hunk 2]
```diff
@@ -86,16 +86,24 @@ If the icon is not filled white, then the file is not available offline.
 If the file is filled white, then it is available offline. 
 Click the button to make the file available, or not available offline.
 
-Sharing Files
-~~~~~~~~~~~~~
+If you set your file or folder as available offline that will keep it synchronized if any change is made in other client.
```
Comment (settermjd) [inline user_manual/ios_app.rst]: Almost spot on. How about the following instead:

> If you set your file or folder as available offline that will keep it synchronized if changes are made in other clients.

File: user_manual/ios_app.rst [hunk 3]
```diff
@@ -86,16 +86,24 @@ If the icon is not filled white, then the file is not available offline.
...
 Click the button to make the file available, or not available offline.
 
-Sharing Files
-~~~~~~~~~~~~~
+If you set your file or folder as available offline that will keep it synchronized if any change is made in other client. 
+
+Your files will always be downloaded and updated.
+
+Sharing Files with users or by public link
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (settermjd) [inline user_manual/ios_app.rst]: The header underline needs to be as long as the header text above it.

File: user_manual/ios_app.rst [hunk 4]
```diff
@@ -86,16 +86,24 @@ If the icon is not filled white, then the file is not available offline.
...
 
 The triangular Share button opens a share dialog, allowing you to start
-`Sharing Files`_. 
+`Sharing Files`_.
+
+You can share with other ownCloud users and edit the privileges for each user.
+
+Multiple public link are supported, you can create multiple links with different options (password, expiration date,..) for each one.
```
Comment (settermjd) [inline user_manual/ios_app.rst]: s/Multiple public link/Multiple public links/g

File: user_manual/ios_app.rst (hunk 1 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 2 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 3 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

File: user_manual/ios_app.rst (hunk 4 above)
Comment (nasli) [inline user_manual/ios_app.rst]: 👍

**Comment by nasli** [comment]
Changes done, Thanks @settermjd !!

**Comment by settermjd** [comment]
Not a problem at all.

**Pull request merged**